### DISCUSSION TITLE
Show layers in the timeline *ONLY* tagged with a specific color(s)

### DISCUSSION THREAD

**Discussion by ota-works** [keiwando/toonsquid#362] [Ideas]
Hi, I had this suggestion in mind for  most of us who are not fond of seeing many layers in the timeline. 
Could you add a feature where we tag specific layers in the timeline with a specific color or keyword such that if we wanted to only view Orange layers in the timeline (not affecting what we see in the display), and only focus on Orange layers. Then if we were satisfied, then we would switch back to regular viewing of all layers.

Maybe we could select multiple layer colors that we want to see in the timeline, like Blue, Red, and Orange, and hides the other untagged or other colors not selected to view.

I think this will go a long way in making layer management very easy for all of us.

**Comment by yaknepper** [comment]
Awesome suggestion. I hope the dev will implement something like this.

**Reply by ota-works** [reply]
Thanks, dude! Seeing all those layers in the Timeline can get quite frustrating, especially when you're working on a project with multiple layers. I believe this feature could really help us enjoy animating without constantly having to scroll through various layers on the timeline
